editors/code/src/main.ts

```diff
@@ -62,11 +66,19 @@ async function tryActivate(_context: vscode.ExtensionContext) {
  * @param platform The detected platform.
  */
 function findCommand(platform: Platform): string | undefined {
+    const exe = `rune-languageserver${platform.ext}`;
     if (!!process.env.RUNE_DEBUG_FOLDER) {
-        return path.join(process.env.RUNE_DEBUG_FOLDER, `rune-languageserver${platform.ext}`);
+        const p = path.join(process.env.RUNE_DEBUG_FOLDER, exe);
+        if (existsSync(p)) { return p; }
+        else {
+            log.warn('env var `RUNE_DEBUG_FOLDER` is set but', p, 'does not exist');
+        }
     }
 
-    console.debug("Cannot find a command for the Rune Language Server.");
+    const p = path.join(process.env.HOME || '~', '.cargo', 'bin', exe);
```

Comment:
Neat! Possibly use [`os.homedir()`](https://nodejs.org/api/os.html#os_os_homedir) instead? I don't believe tilde expansion works for node on Windows unfortunately, and it correctly resolves to `USERPROFILE` where cargo home is by default on Windows.